src/docfx/build/page/ConceptualModel.cs

```diff
@@ -18,7 +18,10 @@ internal class ConceptualModel
         public string? Title { get; set; }
 
         [JsonProperty("rawTitle")]
-        public string? RawTitle { get; set; }
+        public string RawTitle { get; set; } = "";
```

Comment:
seems like we can just remove `SnakeCaseNamingStrategy`, Camel case should be default